src/components/appLayout/HeaderContent.tsx

```diff
@@ -0,0 +1,24 @@
+import { MenuFoldOutlined, MenuUnfoldOutlined } from '@ant-design/icons';
+import { Button } from 'antd';
+import * as React from 'react';
+
+type HeaderContentProps = {
+  isMenuFolded: boolean;
+  setIsMenuFolded: React.Dispatch<React.SetStateAction<boolean>>;
+};
+
+const HeaderContent: React.FC<HeaderContentProps> = (props: HeaderContentProps) => {
+  return (
+    <>
+      <span style={{ paddingLeft: 10, paddingRight: 10 }}>
+        <Button
+          icon={props.isMenuFolded ? <MenuUnfoldOutlined /> : <MenuFoldOutlined></MenuFoldOutlined>}
```

Comment:
```suggestion
          icon={props.isMenuFolded ? <MenuUnfoldOutlined /> : <MenuFoldOutlined />}
```

Be consistent in how you close tags if they have no children